src/main/java/mate/academy/bookstore/dto/order/UpdateOrderRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.bookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.bookstore.model.Order;
+
+@Data
+public class UpdateOrderRequestDto {
+    @NotNull
+    @Size(min = 1, max = 255)
```

Comment:
Order.Status is an enum so it has fixed values. There is no need to validate size in this case.